Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -257,9 +286,12 @@ script:
     - contextPath: Gmail.Body
       description: The content of the email.
       type: string
-    - contextPath: Gmail.Attachments
```

Comment:
I think it can break BC.
Add the new outputs as an addition to `Gmail.Attachments`.